### PR TITLE
Use orgs.Export for contact exports

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: temba/contacts/models.py [hunk 1]
```diff
@@ -1864,13 +1869,37 @@ class ExportContactsTask(BaseExport):
 
     search = models.TextField(null=True, blank=True, help_text=_("The search query"))
 
+    def write_export(self):  # pragma: no cover
```
Comment (rowanseymour) [inline temba/contacts/models.py:1872]: I see you're already mocking this in the tests so I don't think you need this

File: temba/contacts/models.py [hunk 2]
```diff
@@ -1930,25 +1959,31 @@ def get_export_fields_and_schemes(self):
...
-        group = self.group or self.org.active_contacts_group
+    def write(self, export):
+        fields, scheme_counts, group_fields = self.get_export_fields_and_schemes(export)
+        group_id = export.config.get("group_id")
+        search = export.config.get("search")
+
+        if group_id:
+            group = export.org.groups.filter(id=group_id).get()
+        else:
+            group = export.org.groups.get(group_type=ContactGroup.TYPE_DB_ACTIVE)
```
Comment (rowanseymour) [inline temba/contacts/models.py:1975]: `group = export.org.active_contacts_group`

File: temba/orgs/models.py [hunk 3]
```diff
@@ -1699,7 +1702,7 @@ def get_contact_fields(self):
     def get_contact_groups(self):
         ids = self.config.get("with_groups", [])
         id_by_order = {id: i for i, id in enumerate(ids)}
-        return sorted(self.org.groups.filter(id__in=ids), key=lambda o: id_by_order[o.id])
+        return sorted(self.org.groups.filter(id__in=ids).using("readonly"), key=lambda o: id_by_order[o.id])
```
Comment (rowanseymour) [inline temba/orgs/models.py:1705]: not sure how important this is but we should probably make `get_contact_fields` do the same

**Review by rowanseymour — CHANGES_REQUESTED** [review]

File: temba/orgs/tasks.py [hunk 4]
```diff
@@ -79,11 +78,11 @@ def resume_failed_tasks():
     now = timezone.now()
     window = now - timedelta(hours=1)
 
-    contact_exports = ExportContactsTask.objects.filter(modified_on__lte=window).exclude(
-        status__in=[ExportContactsTask.STATUS_COMPLETE, ExportContactsTask.STATUS_FAILED]
+    contact_exports = Export.objects.filter(export_type=ContactExport.slug, modified_on__lte=window).exclude(
```
Comment (rowanseymour) [inline temba/orgs/tasks.py:81]: We should let this be for all exports using `Export` - so don't filter on `export_type`

Comment (rowanseymour) [reply temba/orgs/tasks.py:81]: Let's rename task as well to `restart_stalled_exports`

**Review by rowanseymour — CHANGES_REQUESTED** [review]

File: temba/orgs/tasks.py (hunk 4 above)
Comment (norkans7) [reply temba/orgs/tasks.py]: Update that however I do not see that task in the `CELERY_BEAT_SCHEDULE`,

I guess we never enabled that to run

Comment (rowanseymour) [reply temba/orgs/tasks.py]: Ahh remember the great import fiasco.. hmm well leave it like that for now and let's discuss

**Review by rowanseymour — APPROVED** [review]

**Pull request merged**